src/components/Dashboard/SortList/SortList.jsx

```diff
@@ -0,0 +1,123 @@
+import React, { Component } from 'react';
+import {
+  Dropdown,
+  ButtonGroup,
+  Button,
+} from 'react-bootstrap';
+import storage from './storage';
+import './SortList.scss';
+import { ReactComponent as SortIcon } from './sort.svg';
+import { ReactComponent as SortIconAsc } from './sort-asc.svg';
+import { ReactComponent as SortIconDesc } from './sort-desc.svg';
```

Comment:
Wouldn't it be easier simply store all three svg elements in a single file and export them alltogether?